lib/kitchen.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'menu'
+require_relative 'takeaway'
+
+class Kitchen
+  attr_reader :order_cart
+  attr_accessor :order_complete
+
+  def initialize
+    @menu = Menu.new
+    @order_total = []
+    @order_cart = []
+  end
+
+  def order(order)
```

Comment:
consider calling the parameter menu_item rather than order as the method so it reads better